content/en/network_monitoring/devices/netflow.md

```diff
@@ -43,6 +43,27 @@ network_devices:
 
 After saving your changes, [restart the Agent][4].
 
+## Aggregation
+
+The Datadog Agent will automatically aggregate the received NetFlow data in order to limit the number of records sent to platform while maintaining most of the information. By default there is a 5 minutes aggregation interval, flow recording sharing the same identifying information (source and destination address and port, protocol etc.) will be aggregated together. Addtionnally the Datadog Agent can detect ephemeral port and remove them, this is why you may see Flows with `port:*`.
+
+## Enrichment
+
+Your NetFlow data is processed by the Datadog backend and they are enriched with the available metadata we have about your devices and interfaces. Enrichment is based on the NetFlow exporter IP and the interface indexes. In case of possible private IP collisions, a `namespace` configured on the Agent side is also used.
```

Comment:
```suggestion
Your NetFlow data is processed by the Datadog backend and are enriched with the available metadata from your devices and interfaces. Enrichment is based on the NetFlow exporter IP and the interface indexes. In case of possible private IP collisions, a `namespace` configured on the Agent side is also used.
```